### ISSUE TITLE
How can I apply this algorithm for detector with focal loss

### ISSUE THREAD

**Issue by lichengwei-code** [NVlabs/AL-MDN#1]
Hi, I am very interested in your work. So I wanna apply this algorithm for my work. As a commonly used loss function focal loss， the output of clssification is different from the cross entropy loss. The classification output layer num is equal to the class num, not class num add 1, So I wanna know how to change the loss function in this paper for applying focal loss output. Thanks a lot.

**Comment by jwchoi384** [comment]
Hello, 
I have known that focal loss in their paper (https://arxiv.org/pdf/1708.02002.pdf) is based on a cross-entropy loss with a modulating factor.
I think you need to add a modulating factor that takes into account Gaussian mixing parameters to our proposed loss function.

**Comment by Shanyaodedanshen** [comment]
> Hello, I have known that focal loss in their paper (https://arxiv.org/pdf/1708.02002.pdf) is based on a cross-entropy loss with a modulating factor. I think you need to add a modulating factor that takes into account Gaussian mixing parameters to our proposed loss function.

"add a modulating factor that takes into account Gaussian mixing parameters to our proposed loss function." Yes, I agree with it. It is also recommended that the project "pod-compare", "probdet" in github.
